ui/apps/platform/package.json

```diff
@@ -191,7 +202,8 @@
         "react-dom": "^18.2.0",
         "connected-react-router": {
             "react": "^18.0.0"
-        }
+        },
+        "react-dropzone": "14.3.2"
```

Comment:
The console SDK has a dependency on PatternFly 6, which brings in `react-dropzone@14.3.8`. Versions higher than `14.3.2` have an accessibility issue that fails our tests (good job cypress-axe) due to this issue github.com/react-dropzone/react-dropzone/issues/1430.

Since we are not even using PF6, neither in the main app nor the plugin yet, and since it is somewhat unlikely that PF6 truly depends on >14.3.2 for functionality, we will use an npm override here.